.github/workflows/build.yaml

```diff
@@ -0,0 +1,73 @@
+name: Build
+
+on:
+  push:
+    branches: [ master ]
+  pull_request: 
```

Comment:
No, you're right. I'm going to remove this, because the workflow should only be triggered:
- when pushing any change to the master branch
- regularly using the cron job